core/trino-spi/src/main/java/io/trino/spi/security/SystemAccessControl.java

```diff
@@ -271,7 +275,10 @@ default void checkCanRenameSchema(SystemSecurityContext context, CatalogSchemaNa
      * Check if identity is allowed to change the specified schema's user/role.
      *
      * @throws AccessDeniedException if not allowed
+     *
+     * @deprecated {Use {@link #checkCanSetEntityAuthorization}
      */
+    @Deprecated
     default void checkCanSetSchemaAuthorization(SystemSecurityContext context, CatalogSchemaName schema, TrinoPrincipal principal)
     {
         denySetSchemaAuthorization(schema.toString(), principal);
```

Comment:
Let's switch all of these to `denySetEntityAuthorization`

Reply:
Switched.